packages/financial-templates-lib/src/price-feed/StLouisFedGovPriceFeed.ts

```diff
@@ -46,6 +49,20 @@ export class StLouisFedGovPriceFeed extends PriceFeedInterface {
     };
   }
 
+  public getTimestampInTimeZone = (timeZone: string) => {
```

Comment:
In this function, we are dividing the time by 1000 i.e. converting ms to seconds, right?

but, in `getTimeDifferenceInSeconds` function we are subtracting stLouisTime (i.e. in seconds) from machineTime (i.e. in milliseconds)

@RojhatToptamus This looks incorrect to me?

Reply:
Both values are in seconds. The function below is passed as an argument to the constructor and used to obtain machine time.

`const getTime = () => Math.round(new Date().getTime() / 1000);`

Reply:
Got it, thanks for explaining!